src/CVNBot/ListManager.cs

```diff
@@ -72,11 +72,22 @@ void CollectGarbage(object stateInfo)
                 {
                     lock (dbtoken)
                     {
-                        timcmd.CommandText = "DELETE FROM users WHERE ((expiry < '" + DateTime.Now.Ticks.ToString() + "') AND (expiry != '0'))";
+                        timcmd.CommandText = "DELETE FROM users WHERE((expiry < @UsersExpiryDateTime) AND (expiry != '0'))";
+                        timcmd.Parameters.Add(new SqliteParameter("@UsersExpiryDateTime", DateTime.Now.Ticks.ToString()));
+                        timcmd.Prepare();
                         total += timcmd.ExecuteNonQuery();
-                        timcmd.CommandText = "DELETE FROM watchlist WHERE ((expiry < '" + DateTime.Now.Ticks.ToString() + "') AND (expiry != '0'))";
+
+                        //Clean out parameters list for the next statement
+                        timcmd.Parameters.Clear();
+                        timcmd.CommandText = "DELETE FROM watchlist WHERE ((expiry < @WatchlistExpiryDateTime) AND (expiry != '0'))";
+                        timcmd.Parameters.Add(new SqliteParameter("@WatchlistExpiryDateTime", DateTime.Now.Ticks.ToString()));
```

Comment:
Do these each need a unique parameter name because they re-use the `timcmd` object, or would it be fine to use something like `@expiry` or `@datetime` for all?

Reply:
You don't need unique parameters for each and can simply use standard names like  @expiry or @datetime for all the queries,  especially since the code is emptying out the Parameters List before every SQL statement. Looking back at the code, I'm not sure why I did that in the first place and made things inconsistent with the rest.